php/php.editor/src/org/netbeans/modules/php/editor/completion/PHPCodeCompletion.java

```diff
@@ -1073,7 +1061,61 @@ private void autoCompleteInClassContext(
                     }
                 }
             }
+        } else if (completeFieldTypes(tokenSequence, caretOffset, th, info.getSnapshot().getSource().getFileObject())){
+            autoCompleteFieldType(completionResult, request);
+        }
+    }
+
+    private void autoCompleteFieldType(final PHPCompletionResult completionResult, CompletionRequest request) {
+        // PHP 7.4 Typed Properties 2.0
```

Comment:
Nitpick: maybe we could provide a link to the RFC directly here.

Reply:
Agree with you :)